runtime/src/lib.rs

```diff
@@ -360,6 +360,15 @@ pub fn handle_mailbox_commands(drivers: &mut Drivers) -> ! {
     caliptra_drivers::report_boot_status(RtBootStatus::RtReadyForCommands.into());
     loop {
         enter_idle(drivers);
+        if drivers.mbox.cmd_busy() {
+            let reset_reason = drivers.soc_ifc.reset_reason();
+            if reset_reason == ResetReason::WarmReset {
+                let mut result = DisableAttestationCmd::execute(drivers);
```

Comment:
I think it would be good to add something to the [FwInfo command](https://github.com/chipsalliance/caliptra-sw/blob/b04acd1b50330379971ba084e5afea49f39a1de8/runtime/src/info.rs#L12) which says whether attestation is currently disabled. This will help callers troubleshoot what is wrong.

Reply:
Done